From 35365c44bd4026ea51f623b310b970c7eba58234 Mon Sep 17 00:00:00 2001
From: "Gopal, Puunithaaraj" <puunithaaraj.gopal@intel.com>
Date: Mon, 3 Sep 2018 09:11:11 +0700
Subject: [PATCH 2105/2313] media: i2c: crlmodule-lite: Remove op_sys_clock

There are multiple entries of op_sys_clock. Removing
the duplicates and keep 1 entry of information.

Change-Id: Ia4d53243e6522be870b972fb66eff0de37f736ce
Signed-off-by: Gopal, Puunithaaraj <puunithaaraj.gopal@intel.com>
---
 drivers/media/i2c/crlmodule-lite/crl_adv7481_configuration.h      | 4 ----
 drivers/media/i2c/crlmodule-lite/crl_adv7481_cvbs_configuration.h | 4 ----
 drivers/media/i2c/crlmodule-lite/crl_adv7481_eval_configuration.h | 4 ----
 drivers/media/i2c/crlmodule-lite/crl_adv7481_hdmi_configuration.h | 4 ----
 drivers/media/i2c/crlmodule-lite/crlmodule-sensor-ds.h            | 1 -
 5 files changed, 17 deletions(-)

diff --git a/drivers/media/i2c/crlmodule-lite/crl_adv7481_configuration.h b/drivers/media/i2c/crlmodule-lite/crl_adv7481_configuration.h
index 130dc91..4ebe854 100644
--- a/drivers/media/i2c/crlmodule-lite/crl_adv7481_configuration.h
+++ b/drivers/media/i2c/crlmodule-lite/crl_adv7481_configuration.h
@@ -272,8 +272,6 @@ static struct crl_sensor_detect_config adv7481_sensor_detect_regset[] = {
 	},
 };
 
-static const s64 adv7481_op_sys_clock[] =  {400000000, };
-
 static struct crl_pll_configuration adv7481_pll_configurations[] = {
 	{
 		.input_clk = 24000000,
@@ -607,7 +605,6 @@ static struct crl_ctrl_data adv7481_ctrls[] = {
 		.type = CRL_CTRL_TYPE_MENU_INT,
 		.data.int_menu.def = 0,
 		.data.int_menu.max = ARRAY_SIZE(adv7481_pll_configurations) - 1,
-		.data.int_menu.menu = adv7481_op_sys_clock,
 		.flags = 0,
 		.impact = CRL_IMPACTS_NO_IMPACT,
 		.regs_items = 0,
@@ -686,7 +683,6 @@ static struct crl_sensor_configuration adv7481_crl_configuration = {
 
 	.pll_config_items = ARRAY_SIZE(adv7481_pll_configurations),
 	.pll_configs = adv7481_pll_configurations,
-	.op_sys_clk = adv7481_op_sys_clock,
 
 	.modes_items = ARRAY_SIZE(adv7481_modes),
 	.modes = adv7481_modes,
diff --git a/drivers/media/i2c/crlmodule-lite/crl_adv7481_cvbs_configuration.h b/drivers/media/i2c/crlmodule-lite/crl_adv7481_cvbs_configuration.h
index 7e00727..5ea1f7c 100644
--- a/drivers/media/i2c/crlmodule-lite/crl_adv7481_cvbs_configuration.h
+++ b/drivers/media/i2c/crlmodule-lite/crl_adv7481_cvbs_configuration.h
@@ -57,7 +57,6 @@ static struct crl_register_write_rep adv7481_cvbs_streamoff_regs[] = {
 	{0xC1, CRL_REG_LEN_08BIT, 0x3B, 0x90},
 };
 
-static const s64 adv7481_cvbs_op_sys_clock[] =  {130000000};
 
 static struct crl_pll_configuration adv7481_cvbs_pll_configurations[] = {
 	{
@@ -82,7 +81,6 @@ static struct crl_pll_configuration adv7481_cvbs_pll_configurations[] = {
 		.pll_regs_items = 0,
 		.pll_regs = NULL,
 	 },
-
 };
 
 static struct crl_subdev_rect_rep adv7481_cvbs_ntsc_rects[] = {
@@ -176,7 +174,6 @@ static struct crl_ctrl_data adv7481_cvbs_ctrls[] = {
 		.type = CRL_CTRL_TYPE_MENU_INT,
 		.data.int_menu.def = 0,
 		.data.int_menu.max = ARRAY_SIZE(adv7481_cvbs_pll_configurations) - 1,
-		.data.int_menu.menu = adv7481_cvbs_op_sys_clock,
 		.flags = 0,
 		.impact = CRL_IMPACTS_NO_IMPACT,
 		.regs_items = 0,
@@ -245,7 +242,6 @@ static struct crl_sensor_configuration adv7481_cvbs_crl_configuration = {
 
 	.pll_config_items = ARRAY_SIZE(adv7481_cvbs_pll_configurations),
 	.pll_configs = adv7481_cvbs_pll_configurations,
-	.op_sys_clk = adv7481_cvbs_op_sys_clock,
 
 	.modes_items = ARRAY_SIZE(adv7481_cvbs_modes),
 	.modes = adv7481_cvbs_modes,
diff --git a/drivers/media/i2c/crlmodule-lite/crl_adv7481_eval_configuration.h b/drivers/media/i2c/crlmodule-lite/crl_adv7481_eval_configuration.h
index a0cd825..575d2db 100644
--- a/drivers/media/i2c/crlmodule-lite/crl_adv7481_eval_configuration.h
+++ b/drivers/media/i2c/crlmodule-lite/crl_adv7481_eval_configuration.h
@@ -8,7 +8,6 @@
 
 #include "crlmodule-sensor-ds.h"
 
-static const s64 adv7481_eval_op_sys_clock[] =  {400000000, };
 
 struct crl_ctrl_data_pair ctrl_data_lanes[] = {
 	{
@@ -395,7 +394,6 @@ static struct crl_ctrl_data adv7481_eval_ctrls[] = {
 		.type = CRL_CTRL_TYPE_MENU_INT,
 		.data.int_menu.def = 0,
 		.data.int_menu.max = ARRAY_SIZE(adv7481_eval_pll_configurations) - 1,
-		.data.int_menu.menu = adv7481_eval_op_sys_clock,
 		.flags = 0,
 		.impact = CRL_IMPACTS_NO_IMPACT,
 		.regs_items = 0,
@@ -478,7 +476,6 @@ static struct crl_sensor_configuration adv7481_eval_crl_configuration = {
 
 	.pll_config_items = ARRAY_SIZE(adv7481_eval_pll_configurations),
 	.pll_configs = adv7481_eval_pll_configurations,
-	.op_sys_clk = adv7481_eval_op_sys_clock,
 
 	.modes_items = ARRAY_SIZE(adv7481_eval_modes),
 	.modes = adv7481_eval_modes,
@@ -514,7 +511,6 @@ static struct crl_sensor_configuration adv7481b_eval_crl_configuration = {
 
 	.pll_config_items = ARRAY_SIZE(adv7481_eval_pll_configurations),
 	.pll_configs = adv7481_eval_pll_configurations,
-	.op_sys_clk = adv7481_eval_op_sys_clock,
 
 	.modes_items = ARRAY_SIZE(adv7481_eval_modes),
 	.modes = adv7481_eval_modes,
diff --git a/drivers/media/i2c/crlmodule-lite/crl_adv7481_hdmi_configuration.h b/drivers/media/i2c/crlmodule-lite/crl_adv7481_hdmi_configuration.h
index 599ed55..50bf158 100644
--- a/drivers/media/i2c/crlmodule-lite/crl_adv7481_hdmi_configuration.h
+++ b/drivers/media/i2c/crlmodule-lite/crl_adv7481_hdmi_configuration.h
@@ -225,8 +225,6 @@ static struct crl_register_write_rep adv7481_hdmi_streamoff_regs[] = {
 	{0xC1, CRL_REG_LEN_08BIT, 0x3B, 0x94},
 };
 
-static const s64 adv7481_hdmi_op_sys_clock[] =  {297000000, 445500000};
-
 static struct crl_pll_configuration adv7481_hdmi_pll_configurations[] = {
 	{
 		.input_clk = 24000000,
@@ -590,7 +588,6 @@ static struct crl_ctrl_data adv7481_hdmi_ctrls[] = {
 		.type = CRL_CTRL_TYPE_MENU_INT,
 		.data.int_menu.def = 0,
 		.data.int_menu.max = ARRAY_SIZE(adv7481_hdmi_pll_configurations) - 1,
-		.data.int_menu.menu = adv7481_hdmi_op_sys_clock,
 		.flags = 0,
 		.impact = CRL_IMPACTS_NO_IMPACT,
 		.regs_items = 0,
@@ -663,7 +660,6 @@ static struct crl_sensor_configuration adv7481_hdmi_crl_configuration = {
 
 	.pll_config_items = ARRAY_SIZE(adv7481_hdmi_pll_configurations),
 	.pll_configs = adv7481_hdmi_pll_configurations,
-	.op_sys_clk = adv7481_hdmi_op_sys_clock,
 
 	.modes_items = ARRAY_SIZE(adv7481_hdmi_modes),
 	.modes = adv7481_hdmi_modes,
diff --git a/drivers/media/i2c/crlmodule-lite/crlmodule-sensor-ds.h b/drivers/media/i2c/crlmodule-lite/crlmodule-sensor-ds.h
index b2021ae1..dd98df8 100644
--- a/drivers/media/i2c/crlmodule-lite/crlmodule-sensor-ds.h
+++ b/drivers/media/i2c/crlmodule-lite/crlmodule-sensor-ds.h
@@ -489,7 +489,6 @@ struct crl_sensor_configuration {
 
 	const unsigned int pll_config_items;
 	const struct crl_pll_configuration *pll_configs;
-	const s64 *op_sys_clk;
 
 	const unsigned int modes_items;
 	const struct crl_mode_rep *modes;
-- 
2.7.4

